components/Dashboard/Cards/DashboardCard.js

```diff
@@ -1,21 +1,18 @@
 import PropTypes from 'prop-types';
-import { Paper, Box, Typography, Grid, Hidden } from '@material-ui/core';
+import { Box, Typography, Grid, Hidden } from '@material-ui/core';
 import { useTheme } from '@material-ui/core/styles';
-
+import Card from '../../Shared/Card';
```

Comment:
don't forget to put a space between external package imports and internal file imports